src/parcsr_ls/par_amg_setup.c

```diff
@@ -8,7 +8,7 @@
 #include "_hypre_onedpl.hpp"
 #include "_hypre_parcsr_ls.h"
 #include "par_amg.h"
-#include "../parcsr_block_mv/par_csr_block_matrix.h"
+#include "../parcsr_block_mv/_hypre_parcsr_block_mv.h"
```

Comment:
```suggestion
#include "_hypre_parcsr_block_mv.h"
```

Minor suggestion... Works since we have `-I$(src_dir)/../parcsr_block_mv` in the Makefile

Reply:
Fixed.  See other comment for par_cycle.c.